voice/transcription.go

```diff
@@ -92,3 +92,16 @@ func (trans *Transcription) Contents(client *messagebird.Client) (string, error)
 	b, err := ioutil.ReadAll(resp.Body)
 	return string(b), err
 }
+
+// CreateTranscription creates a transcription request for an existing recording
+func CreateTranscription(client *messagebird.Client, callID string, legID string, recordingID string) (trans *Transcription, err error) {
+	var body struct{}
```

Comment:
Cant you just pass `nil` instead of an empty struct ?

Reply:
API throws error `16` on `nil`:
> The request body could not be parsed. Check if the body contents correspond with the supplied Content-Type HTTP header.

According to https://developers.messagebird.com/docs/voice-calling/troubleshooting/#voice-error-codes